components/movementsensor/replay/replay.go

```diff
@@ -46,12 +48,39 @@ var (
 	// model is the model of a replay movement sensor.
 	model = resource.DefaultModelFamily.WithModel("replay")
 
+	// initializePropertiesTimeout defines the amount of time we allot to the attempt to initialize Properties.
+	initializePropertiesTimeout = 10 * time.Second
+
 	// ErrEndOfDataset represents that the replay sensor has reached the end of the dataset.
 	ErrEndOfDataset = errors.New("reached end of dataset")
 
+	// errPropertiesFailedToInitialize represents that the properties failed to initialize.
+	errPropertiesFailedToInitialize = errors.New("Properties failed to initialize")
+
 	// errCloudConnectionFailure represents that the attempt to connect to the cloud failed.
 	errCloudConnectionFailure = errors.New("failure to connect to the cloud")
 
+	// errSessionClosed represents that the session has ended.
+	errSessionClosed = errors.New("session closed")
+
+	// errPositionNotSupported represents that the Position endpoint does not provide any data.
+	errPositionNotSupported = errors.New("Position is not supported")
```

Comment:
These should be ErrMethodUnimplementedPosition and the like from https://github.com/viamrobotics/rdk/blob/7008506de387b8bfe9e991f25814c6a28a48890f/components/movementsensor/utils.go#L57 so that Readings in `movement_sensor.go` will not report and error when pinging endpoints without data.